web/src/components/Program.js

```diff
@@ -75,17 +76,17 @@ const Program = ({location, title, speaker, brief, streamLink}) => {
         <Title>{title}</Title>
         <Speaker>{speaker}</Speaker>
         <Brief>{brief}</Brief>
-        <Link><a href={streamLink} target="_blank" rel="noreferrer">前往議程直播 →</a></Link>
+          <Link><a href={streamLink} target="_blank" rel="noreferrer">前往議程直播 →</a></Link>
       </Context>
     </Container>
   )
 };
 
 Program.defaultProps = {
   location: "RX",
-  title: "你所未知的世界",
-  speaker: "不知道哪裡來的人",
-  brief: "超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹超級簡短的介紹",
+  title: "沒有進行中的議程",
```

Comment:
oh, you don't need to change this, this is a default value to display component layout